portfolio.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="portfolio.css" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.1/css/all.css" />
+    <title>My Portfolio</title>
+</head>
+
+<body>
+    <header>
+        <nav class="topnav">
+            <div>
+                <a class="mylogo" href="#mylogo">My Logo</a>
+            </div>
+            <ul class="nav__links">
```

Comment:
- [ ] I strongly suggest you to uphold the figma design. I noticed that the `humburger` button looks totally different. Lets try to match the style in your project with the one provided in figma. Please decrease its size and set the colour of `humburger` and the logo to `#6070FF`

Your design  | Figma design
------------- | -------------
<img width="377" alt="Screenshot 2022-02-10 at 21 04 05" src="https://user-images.githubusercontent.com/77038610/153458661-955e5d1c-4de4-4f96-b7db-9f48bb6b88e9.png">  | <img width="376" alt="Screenshot 2022-02-10 at 21 04 53" src="https://user-images.githubusercontent.com/77038610/153458812-4ff744c5-dcfd-4866-8e96-b7ea3f05a918.png">